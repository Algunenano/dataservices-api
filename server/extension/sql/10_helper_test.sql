-- Create a conf table
CREATE TABLE conf (key TEXT PRIMARY KEY, values_json TEXT NOT NULL);
INSERT INTO conf VALUES ('cds', '{"Manolo Escobar": {"El Limonero":"En stock", "Viva el vino":"Sin stock"}}');

-- Test key retrieval
SELECT cdb_geocoder_server._get_conf('cds');

-- Test no key exception
SELECT cdb_geocoder_server._get_conf('no existe');

-- Drop conf table
DROP TABLE conf;
