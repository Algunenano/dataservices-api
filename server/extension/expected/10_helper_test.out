-- Create a conf table
CREATE TABLE conf (key TEXT NOT NULL PRIMARY KEY, values_json TEXT);
INSERT INTO conf VALUES ('cds', '{"Manolo Escobar": {"El Limonero":"En stock", "Viva el vino":"Sin stock"}}');
-- Test key retrieval
SELECT cdb_geocoder_server._get_conf('cds');
                                 _get_conf                                  
----------------------------------------------------------------------------
 {"Manolo Escobar": {"El Limonero":"En stock", "Viva el vino":"Sin stock"}}
(1 row)

-- Test no key exception
SELECT cdb_geocoder_server._get_conf('no existe');
ERROR:  Missing key 'no existe' in conf table
-- Drop conf table
DROP TABLE conf;
